eZ/Publish/API/Repository/URLAliasService.php

```diff
@@ -118,11 +137,18 @@ public function lookup($url, $languageCode = null);
      * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if no url alias exist for the given language
      *
      * @param \eZ\Publish\API\Repository\Values\Content\Location $location
-     * @param string $languageCode
+     * @param string|null $languageCode
+     * @param bool|null $showAllTranslations
+     * @param string[]|null $prioritizedLanguages
      *
      * @return \eZ\Publish\API\Repository\Values\Content\URLAlias
      */
-    public function reverseLookup(Location $location, $languageCode = null);
+    public function reverseLookup(
+        Location $location,
+        ?string $languageCode = null,
+        ?bool $showAllTranslations = null,
+        ?array $prioritizedLanguages = null
```

Comment:
`listLocationAliases` uses `array $prioritizedLanguages = null` (not nullable with `null` as default value). This should be streamlined but for me they both should be not nullable and use an empty array as a default value.